test/unit/io.js

```diff
@@ -856,6 +866,37 @@ describe('Input/output', function () {
       });
   });
 
+  it('Specifying quantisation table provides different JPEG', function (done) {
+    // First generate with default quantisation table
+    sharp(fixtures.inputJpg)
+      .resize(320, 240)
+      .jpeg({ optimiseCoding: false })
+      .toBuffer(function (err, withDefaultQuantisationTable, withInfo) {
+        if (err) throw err;
+        assert.strictEqual(true, withDefaultQuantisationTable.length > 0);
+        assert.strictEqual(withDefaultQuantisationTable.length, withInfo.size);
+        assert.strictEqual('jpeg', withInfo.format);
+        assert.strictEqual(320, withInfo.width);
+        assert.strictEqual(240, withInfo.height);
+        // Then generate with different quantisation table
+        sharp(fixtures.inputJpg)
+          .resize(320, 240)
+          .jpeg({ optimiseCoding: false, quantisationTable: 3 })
+          .toBuffer(function (err, withKodakTable, withoutInfo) {
```

Comment:
Table 3 was hand-rolled by image processing expert and libvips contributor Nicolas Robidoux rather than tuned via the Kodak set - see http://www.imagemagick.org/discourse-server/viewtopic.php?f=22&t=20333&p=98008#p98008

Reply:
Oops. I guess the quantisation tables might benefit from having constants defined, but I have no idea how those would be named. Anyway, should I just rename it to `withQuantTable3`, or `withImageMagickTable`? Any preferences?

Reply:
Let's go for your `withQuantTable3` option, thanks. Hopefully this will prevent someone (me) in the future (next month) getting confused that there might be an off-by-one error here.

Reply:
I've amended my previous commit to change the variable name, thanks for noticing!